avahi-daemon/avahi-daemon.conf

```diff
@@ -26,6 +26,8 @@ use-ipv4=yes
 use-ipv6=no
 #allow-interfaces=eth0
 #deny-interfaces=eth1
+#allow-addresses=192.168.50.10, 192.168.50.11
```

Comment:
I think if we should not allow also network prefixes here. Like 192.168.50.10/30.